### ISSUE TITLE
jruby.org/documentation page is suffering from link rot

### ISSUE THREAD

**Issue by ikaronen-relex** [jruby/jruby#8065]
While looking for a tutorial on how to set up a basic JRuby demo app with custom Java classes (still haven't found one) I ended up on https://www.jruby.org/documentation and noticed that it could use some TLC.

At least the first three links in the "Recent Articles" section seem to be dead:

- https://www.retroaxis.info/2019/07/10/my-experience-architecting-a-software-development-stack-with-jruby-openjdk-and-roda/ gives an SSL error
- http://www.leaphacking.com/posts/the-pleasures-of-jruby.html is blocked by my company VPN as a parked domain
- http://pragprog.com/magazines/2013-02/deploying-with-jruby-in-the-cloud says "Page Not Found"

(Also, is an article from 2013 really "recent" anymore?)

Both of the links in the "Recent Videos" section seem to be broken too, leading to a page saying "The domain Confreaks.tv may be for sale." I guess those should be removed, too.

**Comment by headius** [comment]
You are correct! There's a lot of content in the wiki and only a couple of us doing updates.

I would be happy to add you if you want to make some changes. We only locked it down because it's not possible to make Google index the wiki if it is publicly editable (Github security restrictions.

We definitely can find more recent articles to put in there, and pretty much every page on the wiki needs an audit.

**Comment by ikaronen-relex** [comment]
@headius Sure, I can fix that if you give me edit rights to the wiki. I won't promise to be a very active editor, but I can fix obvious problems if I run into them.

**Comment by headius** [comment]
@ikaronen-relex I added you to the team with "triage" abilities because it's not clear what level you need to write to the wiki. Let me know if that doesn't work.

**Comment by ikaronen-relex** [comment]
@headius Hmm, still no edit button that I can see. I believe it should be somewhere around here, right?

<img width="1728" alt="Screenshot 2024-01-19 at 17 44 23" src="https://github.com/jruby/jruby/assets/38075778/76a7f740-9f46-4417-8335-cc967d350701">

I even tried logging out and back in. I guess the access level probably isn't enough.

**Comment by ikaronen-relex** [comment]
@headius Hmm, I still don't see an edit button on the [wiki](https://github.com/jruby/jruby/wiki) even after accepting the invitation to join to JRuby team (which I didn't notice in my inbox at first).

However, I _did_ figure out [where the documentation page is maintained](https://github.com/jruby/jruby.github.io/blob/master/documentation.markdown) and managed to create [a PR to fix it](https://github.com/jruby/jruby.github.io/pull/47), so I guess that's accomplished at least. 😄
